src/Post.php

```diff
@@ -1215,25 +1215,26 @@ protected function get_revised_data_from_method($method, $args = false)
      *
      * @param int $page Optional. The page to show if the content of the post is split into multiple
      *                  pages. Read more about this in the [Pagination Guide](https://timber.github.io/docs/v2/guides/pagination/#paged-content-within-a-post). Default `0`.
-     *
-     * @return string
+     * @param int $num_words Optional. The number of words to show. Default `-1` (show all).
+     * @param bool $remove_blocks Optional. Whether to remove blocks. Defaults to false. True when called from the $post->excerpt() method.
+     * @return string The content of the post.
      */
-    public function content($page = 0, $len = -1)
+    public function content($page = 0, $num_words = -1, $remove_blocks = false)
```

Comment:
Just a hint that if we rename parameters, as of PHP 8.0, it should be considered a breaking change, see https://php.watch/versions/8.0/named-parameters and the [Named Parameters in Public APIs](https://php.watch/versions/8.0/named-parameters#:~:text=Named%20Parameters%20in%20Public%20APIs) in particular.

Reply:
Gotcha! Just reverted to the old name.